hyundai_kia_connect_api/Vehicle.py

```diff
@@ -109,6 +105,10 @@ class Vehicle:
 
     # EV fields (EV/PHEV)
 
+
+    ev_charge_limits_dc: int = None
```

Comment:
_Context: You may already know this but Python has "weak" typing, meaning any object can be assigned with any type. Type hinting does not prevent it but at least makes the code more readable, allows tools/IDEs to identify typing errors, and helps prevent human errors._

I'm going to nitpick here, but if an attribute/variable is typed as `int` (or any other type other than `NoneType`), then it should not be `None`.

On the other hand, if the API does not always return the value, I agree we should have a default that indicates that we don't have a value, rather than a default with a wrong value (i.e zero).

To be honest I'm not sure how to handle this (probably very common) case. My advice is to use `typing.Union` to allow both `None` and `int`.

More on Union: https://stackoverflow.com/questions/33945261/how-to-specify-multiple-return-types-using-type-hints

so the syntax would be: 
```
ev_charge_limits_dc: typing.Union[int, None] = None
```

Reply:
We could have None type. We only set vehicle values based on the data we get so if no data received no data set and None remains.  

This also allows us to clearly tell what the car offers in kia_uvo or other use cases as if no value we aren't left guessing if it actually is 0 or if that just the default.  Typing Union looks like the correct way to go.